src/components/PathTable.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <div :style="divStyle">
+    <v-card>
+      <v-row>
+        <v-col class="pl-6">
+          <v-icon @mousedown="iconMouseDown">
+            mdi-drag-variant
+          </v-icon>
+        </v-col>
+        <v-spacer />
+        <v-col class="text-right pr-6">
+          <v-icon @click="closeCard">
+            mdi-close
+          </v-icon>
+        </v-col>
+      </v-row>
+      <v-card
+        color="grey lighten-3"
+        flat
+        tile
+      >
+        <v-card-title class="pt-0 mt-0">
+          <v-text-field
+            v-model="search"
+            append-icon="mdi-magnify"
+            label="Search"
+            single-line
+            hide-details
+          />
+        </v-card-title>
+      </v-card>
+      <v-data-table
+        :headers="headers"
+        :items="tableData"
+        :search="search"
+      >
+        <template
+          v-slot:top
+        >
+          <v-list dense>
+            <v-list-item dense>
+              <v-row>
+                <v-col
+                  v-for="(header, i) in headers"
+                  :key="i"
+                  class="pa-0"
```

Comment:
I think this needs some x padding
```suggestion
                  class="py-0"
```

Reply:
Good suggestion!